docs/development/Offline_mode.md

```diff
@@ -76,3 +76,130 @@ The main problem is deciding what changes should be saved and what changes shoul
 
    
 To accommodate this there might need to add more attributes in the offline and online databases in order to deal with and keep track of the synchronization. Examples could be “last_updated_on”, “created_on”, “deleted_on”, “edited_offline” which are timestamps used to see if data should be synched or not and the “edited_offline” could be a boolean. It is also an option to use UUID with/instead of timestamps to make the synchronization have an unique id. 
+
+
+## Approaches of how to (and not to) implement the offline repository
+
+The [api_client](https://github.com/aau-giraf/api_client) maps json output from the [web-api](https://github.com/aau-giraf/web-api) into models in which the [weekplanner](https://github.com/aau-giraf/weekplanner) uses for displaying data models. Thus, it would be essential to implement the offline repository feature in the api_client. Every model in the api_client implements an abstract class called `Model` which provides a `from_json()` and `to_json()` method for the models to interact with the web-api. 
+
+### NoSQL / Storing plain JSON
+Our first approach was to store the string value of `to_json()` of each model in a local database. With this, you could get all models back from the database with their `from_json()` method. However, this raises a lot of errors when dealing with relational models. Consider this example:
+
+A model `A` has a one-to-many with the model `B`. If an instance of `B`, related to an instance `A`, was to be altered, it would not be altered on `A`.
+
+Thus, storing the plain `to_json()` string values from models is not a valid solution on related models.
+
+
+### In-memory design

```

Comment:
Not really clear what information should be stored in memory. Do you mean all objects from the database?

Reply:
What do you mean by "objects from the database"?. If the sentence is "data retrieved by the web-api", then I do not follow.

If the sentence should be "objects stored in the offline database", then the hole `### In-memory design` section has been misunderstood. 

The purpose of this section is to guide the next students towards a solution for the offline repository. A way of doing this, is to have an in-memory database. Maybe this is not clear enough in the section

Reply:
Well, there is no mention of what and how much data should be stored in memory. I'm sure the idea isn't to store all information from the online database as that is probably unfeasible, but after reading this section, it is not clear what data and how much data should be stored. Or maybe it's just me

Reply:
btw the comment was meant for the entire `### In-memory design` section

Reply:
Ah I see. Well we have no considerations or recommendations on how **much** data to store in the offline repository, since the original issue was to *make* the actual repository, not how it should be used in any context. 

Maybe these thoughts should be added to clarify?

Reply:
Yeah, I was also just thinking of adding one sentence for clarification.